server/plugins/MeshtasticsLoRaAPI.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "MeshtasticsLoRaAPI",
+    "description": "Integration for LoRa, utilising the provided HTTP API. <br>For it to work, you need to specify a chatURL file once on the Host. <br>To do this run: 'meshtastic --set-url https://meshtastic.com/o/thegivenurl  ' on the sender as well as the reciever. ",
```

Comment:
Need to add any pre-requisites the user requires as well, such as the python cli you are using.